dex/src/main/scala/com/wavesplatform/dex/model/OrderBookSnapshot.scala

```diff
@@ -21,17 +22,21 @@ object OrderBookSnapshot {
         dest += 1
         LastTrade.serialize(dest, lastTrade)
     }
+    dest ++= Longs.toByteArray(x.nextTxTimestamp)
   }
 
-  def fromBytes(bb: ByteBuffer): OrderBookSnapshot =
-    OrderBookSnapshot(
-      OrderBookSideSnapshotCodecs.decode(bb),
-      OrderBookSideSnapshotCodecs.decode(bb),
-      bb.get match {
-        case 0 => None
-        case 1 => Some(LastTrade.fromBytes(bb))
-        case x => throw new RuntimeException(s"Can't deserialize Option as $x")
-      }
-    )
+  def fromBytes(bb: ByteBuffer): OrderBookSnapshot = {
+    val bids = OrderBookSideSnapshotCodecs.decode(bb)
+    val asks = OrderBookSideSnapshotCodecs.decode(bb)
+    val lastTrade = bb.get match {
+      case 0 => None
+      case 1 => Some(LastTrade.fromBytes(bb))
+      case x => throw new RuntimeException(s"Can't deserialize Option as $x")
+    }
+    val ts =
+      if (bb.hasRemaining) bb.getLong
```

Comment:
The problem here: we can't add new fields after this change. I know, we should migrate to protobuf or something else, but with such change we limit our decision in the future. A new feature could have a more priority than a migration to other format.

So, let's use a principle that we use in `lastTrade`: 0 - there is no timestamp, 1 - we have `netxtTxTimestamp`